tests/test-sources/plugins/by-name/wrapping/default.nix

```
@@ -0,0 +1,11 @@
+{ pkgs, ... }:
+{
+  empty = {
+    plugins.wrapping.enable = true;
+  };
+
+  defaults.plugins.which-key = {
+    enable = true;
+    notify_on_switch = true;
+  };
```

```suggestion
  defaults = {
    plugins.which-key = {
      enable = true;
      notify_on_switch = true;
    };
  };
```
Purely cosmetic, for consistency with the rest of the code base.